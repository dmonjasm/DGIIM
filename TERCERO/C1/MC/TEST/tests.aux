\relax 
\providecommand\hyper@newdestlabel[2]{}
\@nameuse{bbl@beforestart}
\catcode `"\active 
\catcode `<\active 
\catcode `>\active 
\@nameuse{es@quoting}
\providecommand\HyperFirstAtBeginDocument{\AtBeginDocument}
\HyperFirstAtBeginDocument{\ifx\hyper@anchor\@undefined
\global\let\oldcontentsline\contentsline
\gdef\contentsline#1#2#3#4{\oldcontentsline{#1}{#2}{#3}}
\global\let\oldnewlabel\newlabel
\gdef\newlabel#1#2{\newlabelxx{#1}#2}
\gdef\newlabelxx#1#2#3#4#5#6{\oldnewlabel{#1}{{#2}{#3}}}
\AtEndDocument{\ifx\hyper@anchor\@undefined
\let\contentsline\oldcontentsline
\let\newlabel\oldnewlabel
\fi}
\fi}
\global\let\hyper@last\relax 
\gdef\HyperFirstAtBeginDocument#1{#1}
\providecommand\HyField@AuxAddToFields[1]{}
\providecommand\HyField@AuxAddToCoFields[2]{}
\babel@aux{spanish}{}
\@writefile{toc}{\contentsline {section}{\numberline {1}Test Relación 1}{3}{section.1}\protected@file@percent }
\@writefile{toc}{\contentsline {section}{\numberline {2}Test Relación 2}{6}{section.2}\protected@file@percent }
\@writefile{toc}{\contentsline {section}{\numberline {3}Test Relación 3}{10}{section.3}\protected@file@percent }
\@writefile{toc}{\contentsline {section}{\numberline {4}Test Relación 4}{18}{section.4}\protected@file@percent }
\@writefile{toc}{\contentsline {section}{\numberline {5}Test Relación 5}{21}{section.5}\protected@file@percent }
\@writefile{toc}{\contentsline {section}{\numberline {6}Test Relación 6}{26}{section.6}\protected@file@percent }
\@writefile{toc}{\contentsline {section}{\numberline {7}Ejemplos Pregunta Cortas}{30}{section.7}\protected@file@percent }
