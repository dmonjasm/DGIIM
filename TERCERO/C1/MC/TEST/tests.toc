\babel@toc {spanish}{}
\contentsline {section}{\numberline {1}Test Relación 1}{3}{section.1}%
\contentsline {section}{\numberline {2}Test Relación 2}{6}{section.2}%
\contentsline {section}{\numberline {3}Test Relación 3}{10}{section.3}%
\contentsline {section}{\numberline {4}Test Relación 4}{18}{section.4}%
\contentsline {section}{\numberline {5}Test Relación 5}{21}{section.5}%
\contentsline {section}{\numberline {6}Test Relación 6}{26}{section.6}%
\contentsline {section}{\numberline {7}Ejemplos Pregunta Cortas}{30}{section.7}%
