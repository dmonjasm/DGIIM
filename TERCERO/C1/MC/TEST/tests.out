\BOOKMARK [1][-]{section.1}{Test Relaci\363n 1}{}% 1
\BOOKMARK [1][-]{section.2}{Test Relaci\363n 2}{}% 2
\BOOKMARK [1][-]{section.3}{Test Relaci\363n 3}{}% 3
\BOOKMARK [1][-]{section.4}{Test Relaci\363n 4}{}% 4
\BOOKMARK [1][-]{section.5}{Test Relaci\363n 5}{}% 5
\BOOKMARK [1][-]{section.6}{Test Relaci\363n 6}{}% 6
\BOOKMARK [1][-]{section.7}{Ejemplos Pregunta Cortas}{}% 7
